osutil/group.go

```diff
@@ -29,13 +29,13 @@ import (
 
 // FindUid returns the identifier of the given UNIX user name. It will
 // automatically fallback to use "getent" if needed.
-func FindUid(username string) (uint64, error) {
+var FindUid = func(username string) (uint64, error) {
 	return findUid(username)
 }
```

Comment:
I think this can be simplified to `var FindUid = findUid` and similarly so for the gid below

Reply:
I pushed this change, once green I will merge